### PR TITLE
Fixes modular computer integrity

### PR DESCRIPTION
**Pull request by Shadowlight213**
:cl:
fix: Modular consoles have been fixed.
/:cl:

### CODE REVIEW

**Comment by CyberbossJHCB** [comment]
Missing your cl there, buddy

File: code/modules/modular_computers/computers/item/computer.dm [hunk 1]
```diff
@@ -31,6 +31,7 @@
 	var/steel_sheet_cost = 5								// Amount of steel sheets refunded when disassembling an empty frame of this computer.
 
 	obj_integrity = 100
+	integrity_failure = 50
```
Comment (phil235) [inline code/modules/modular_computers/computers/item/computer.dm]: Tthe item doesn't have a version of obj_break(), so why give it a failure level?

Comment (Shadowlight213) [inline code/modules/modular_computers/computers/item/computer.dm]: Tablets are a child of this.

Comment (Shadowlight213) [inline code/modules/modular_computers/computers/item/computer.dm]: @phil235 
https://github.com/tgstation/tgstation/blob/master/code/modules/modular_computers/computers/item/computer.dm#L181
and
https://github.com/tgstation/tgstation/blob/master/code/modules/modular_computers/computers/item/computer.dm#L195

It does use this value.

Comment (phil235) [inline code/modules/modular_computers/computers/item/computer.dm]: my bad then.

**Pull request merged**